src/operator/regression_output-inl.h

```diff
@@ -129,6 +129,27 @@ class RegressionOutputProp : public OperatorProperty {
     return true;
   }
 
+    bool InferType(std::vector<int> *in_type,
+                 std::vector<int> *out_type,
+                 std::vector<int> *aux_type) const override {
+    CHECK_EQ(in_type->size(), 2) << "Input:[data, label]";
+    int dtype = (*in_type)[0];
+
+    auto nin = in_type->size();
+    in_type->clear();
+    in_type->push_back(dtype);
+    for (index_t i = 1; i < nin; ++i) in_type->push_back(dtype);
```

Comment:
Please follow the coding convention, use {} to wrap it, and put it in a separate line.